components/backup/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "backup"
+version = "0.0.1"
+edition = "2018"
```

Comment:
Is this rust version?

Reply:
Sort of, it enables new features like using crates without `extern crate ...`.